RackApp/app.rb

```diff
@@ -0,0 +1,85 @@
+require 'erb'
+require 'rack'
+require 'active_support/all'
+
+class View
+     attr_reader :person, :time
+    def initialize(user, time)
+        @person = user
+        @time = time
+    end
+
+    def file
+        File.read(File.join(__dir__,"template.html.erb"))
+    end
+
+    def template
+        ERB.new(file)
+    end
+
+    def render
+        template.result(binding)
+    end
+end
+
+class User
+    attr_reader :nickname
+
+    def initialize(nickname)
+        @nickname = nickname
+    end
+    
+end
+
+class CountryTime
+    attr_reader :zone, :timezone
+
+    def initialize(zone)
+       @zone = zone
+   end
+
+   def timezone
+    Time.zone = @zone
+    @time = Time.zone.now
+   end
+end
+
+class WebApp
+    def initialize
+        
+    end
+end
+class Display
+    def self.call(env)
+        req = Rack::Request.new(env)
+        if  req.path.match('/users/hema') && req.params["tz"] == "IST"
```

Comment:
So those are static values, can you make them dynamic?:

- Display what you have in path as the nickname
- Select the Timezone from the `tz` parameter as timezone